tests/languages/fsharp/keyword_feature.test

```diff
@@ -4,19 +4,20 @@ class;
 default
 delegate do done downcast
 downto elif else end
-exception extern false finally
+exception;
+extern false finally
 for fun function global
-if in inherit inline
+if in inherit; inline
 interface;
 internal lazy let
 let! match member module
 mutable namespace
 new;
 not
-null of open or override
+null of; open or override
 private public rec return
 return! select static struct
-then to true try type
+then to true try type;

```

Comment:
Curious about these semicolons here. Are they syntactically correct for F#?

Reply:
To rephrase, were the old code snippets wrong?

Reply:
That's because after `type`, `inherit`, etc. follow class names.
If that semicolon isn't there, the next keyword will be highlighted as a class name.

Reply:
Worth adding a test that demonstrates this? Or do we have one for that?

Reply:
Sure we have! [`class-name_feature.test`](https://github.com/PrismJS/prism/pull/1573/files#diff-5d213637dca63da3247f9c7a7a702697)